### PR TITLE
docs: refactor Accordion readme to the new format

### PR DESCRIPTION
**Pull request by burakukula**

### CODE REVIEW

**Review by massao — APPROVED** [review]

File: packages/components/accordion/examples/AccordionExample.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Accordion, Text } from '@contentful/f36-components';
+
+export default function IconButtonExample() {
+  return (
+    <Accordion>
+      <Accordion.Item title="What payment methods do you accept?">
```
Comment (suevalov) [inline packages/components/accordion/examples/AccordionExample.tsx:7]: Can we have more than one accordion.item in the example?

File: packages/components/accordion/examples/AccordionWithOtherComponentExample.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  Text,
+  SectionHeading,
+  Stack,
+  Button,
+} from '@contentful/f36-components';
+
+export default function IconButtonExample() {
+  return (
+    <Accordion>
+      <Accordion.Item
```
Comment (suevalov) [inline packages/components/accordion/examples/AccordionWithOtherComponentExample.tsx:13]: the same as above

File: packages/components/accordion/examples/AccordionExample.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Accordion, Text } from '@contentful/f36-components';
+
+export default function IconButtonExample() {
+  return (
+    <Accordion>
```
Comment (suevalov) [inline packages/components/accordion/examples/AccordionExample.tsx:6]: is there a way to control which items are opened programmatically? (controlled vs uncontrolled)

**Review by suevalov — CHANGES_REQUESTED** [review]
I think we need to have a bit more sophisticated examples here

**Pull request merged**